src/components/Atoms/Link/Link.js

```diff
@@ -46,7 +53,7 @@ const Link = ({
       underline={underline}
     >
       {children}
-      {target === 'blank' && <HelperText>(opens in new window)</HelperText>}
+      {window === '_blank' && <HelperText>(opens in new window)</HelperText>}
```

Comment:
Might as well tie this into our logic; `target` has the opportunity to be `null` in the prior code